samples/project-anfield/import_cluster_config.py

```diff
@@ -180,7 +183,17 @@ def create_sources(source, environment, node):
     try:
         body = RegisterProtectionSourceParameters()
         body.environment = environment
-        if environment == env_enum.KGENERICNAS:
+        if environment == env_enum.KSQL:
+            name =  node["protectionSource"]["name"]
+            body = RegisterApplicationServersParameters()
+            body.applications = [env_enum.KSQL]
+            id = node["protectionSource"]["id"]
```

Comment:
Why are we adding the id in creating sources ? or this is the parent id ?

Reply:
This is the id of physical source if, which we are going to register as mssql source, also even after registration both physical source and mssql source will have the same source Id.